main.js

```diff
@@ -0,0 +1,18 @@
+const navbarHamburger = document.querySelector('#navbar-hamburger');
+const mobileMenu = document.querySelector('#mobile-menu');
```

Comment:
- [ ] Also here you need to select the right element as you selected it as id but it should be selected as a class.

```javascript
const mobileMenu = document.querySelector('.mobile-menu');
```